### DISCUSSION TITLE
adding support for Glassdoor Jobs?

### DISCUSSION THREAD

**Discussion by RKNGLo** [damianmgarcia/Hide-n-Seek#28] [Ideas]
Hey Damian, absolutely love the application and see it as a must have for apply on LinkedIn

Hey I know that isn't as high traffic as indeed or linkedin, but would it be possible to implement your application on Glassdoor Jobs? 

It even has a similar style of UI at least visually to the two you already cover. 

Thanks 

~Eli

**Comment by damianmgarcia** [comment]
Thank you for your feedback, Eli! I'll create a Glassdoor account this weekend and take a look at it. Take care! ' v '

**Comment by damianmgarcia** [comment]
Hi! I Just wanted to let you know that Glassdoor is now supported. ' v '

**Comment by RKNGLo** [comment]
\(^o^)/ Amazing! Just tested it thank you!
